README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Overview
 
-`libqtxdg` is a Qt 5 implementation of freedesktop.org XDG specifications.   
+`libqtxdg` is a Qt 6 implementation of freedesktop.org XDG specifications.   
```

Comment:
What about "... is a Qt implementation of..."?

Reply:
Preparing for Qt 7, why not.  It came to my mind that we could add instructions for building Qt5 (with `checkout <commit>` ?).

Reply:
The Qt version is obvious from the dependencies. The main info here is that libqtxdg is a *Qt implementation* of XDG specs.